cpp/include/algorithms.hpp

```diff
@@ -621,6 +621,7 @@ void bfs(raft::handle_t const &handle,
  * @param[out] num_level             number of levels of the returned clustering
  * @param[out] clustering            Pointer to device array where the clustering should be stored
  * @param[in]  max_iter              (optional) maximum number of iterations to run (default 100)
+ * @param[in] resolution             The value of the resolution parameter to use
```

Comment:
Can we add a link to the permanent document that describes what this resolution parameter is (or add more description about what this resolution parameter actually does)?

Reply:
I added the link in the python documentation.  Do we want it here also?